src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -146,7 +146,7 @@ interface ExtensionSuggestion {
 
 const extensionPacks: ExtensionSuggestion[] = [
 	{ name: localize('welcomePage.javaScript', "JavaScript"), id: 'dbaeumer.vscode-eslint' },
-	{ name: localize('welcomePage.typeScript', "TypeScript"), id: 'ms-vscode.vscode-typescript-tslint-plugin' },
+	// { name: localize('welcomePage.typeScript', "TypeScript"), id: 'ms-vscode.vscode-typescript-tslint-plugin' },
```

Comment:
Can you please just delete this line instead. No reason to keep it around

Reply:
Sure thing!